filebeat/harvester/reader/log.go

```diff
@@ -28,13 +28,13 @@ type logFileReader struct {
 }
 
 type LogFileReaderConfig struct {
-	CloseOlder         time.Duration
-	BackoffDuration    time.Duration
-	MaxBackoffDuration time.Duration
-	BackoffFactor      int
-	CloseRenamed       bool
-	CloseRemoved       bool
-	CloseEOF           bool
+	Backoff       time.Duration
+	MaxBackoff    time.Duration
+	BackoffFactor int
```

Comment:
I think I like BackoffDuration better, as we've got 3 configs with Backoff here. Alternatively:

```
type BackoffConfig struct {
    Duration time.Duration
    Max time.Duration
    Factor float32
}
```